### ISSUE TITLE
searchable?

### ISSUE THREAD

**Issue by gpedro** [benjaminkomen/TibiaWikiApi#57]
if I want to search an item when it contains "sword" keyword, this API have a search method or something related?

**Comment by benjaminkomen** [comment]
No, it doesn't. I could add such a feature. Would you want to search through only the item names, or also other item properties, like the notes of an item? In the latter case, you would of course also find items which only mention "sword" in the notes, but are not actually a sword.

**Comment by gpedro** [comment]
currently, I use the [TibiaWikiSQL](https://galarzaa90.github.io/tibiawiki-sql/) to generate a JSON to make a client-searchable tool to search by itemid (clientid) and name. but every time I need to generate this JSON file; Check my tool: https://gpedro.github.io/tibia-clientid-finder/ (i will release open-source soon)

I have a project focused on developing an open tibia server: https://github.com/opentibiabr/otservbr-global

That's the reason why I need this tool. Because when we get a raw byte to implement some feature, I need to check if raw bytes matches with some itemid to predict the packet structure.

**Comment by benjaminkomen** [comment]
Ok, thx for explaining the reason why you need this, but I'm still not really sure if you want to:
- search only through item names?
- search through the complete items?

**Comment by gpedro** [comment]
search by item id and item name into all elements of the game (items, creatures, stones, walls, ...)

**Comment by benjaminkomen** [comment]
Closing as wontfix after 2026 triage ([#392](https://github.com/benjaminkomen/TibiaWikiApi/pull/392)). Not planned for this API.
